lib/task-data/schemas/dell-wsman-delete-volume.json

```diff
@@ -0,0 +1,23 @@
+{
+    "copyright": "Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "volumeId": {
+            "description": "Specify volumeId for deleting volume",
+            "type": "string"
+        },
+        "shutdownType": {
+            "description": "Specify the type of system shutdown: Graceful(0), Forced(1)",
+            "type": "integer",
+            "enum": [0,1]
+        }
+    },
+    "properties": {
+        "volumeId": {
+            "$ref": "#/definitions/volumeId"
+        },
+        "shutdownType": {
+            "$ref": "#/definitions/shutdownType"
+        }
+    },
+    "required": ["volumeId"]
```

Comment:
if shutdownType is optional, you'd better tell its default value in its description.

Reply:
And there is difference between "setting a default value in task definition" and "describe it as optional in task schema".  If the job needs this option for execution, then the option has to be `required` no matter you set a default value in task definition or not.

Reply:
I will add description for default value.